README.md

```diff
@@ -9,3 +10,6 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof:
+Because we know that our two graphs of interest have the same number of nodes we know it is at least possible for a function between them to be one-to-one and onto. Because both graphs are fully connected, we also know that each node in $G_1$ is equivalent to each other other node in $G_1$ in how many edges they both have, we know this for $G_2$. We also know that each node in $G_1$ is equivalent to each node in $G_2$ at least with respect to how many edges there are attached to each of them. This means that any mapping with no repetition from each node in $G_1$ to $G_2$ will be one-to-one and onto. A particular function that can map from one graph to the other can be constructed by taking all of the nodes from $G_1$ taking a random permutation of those nodes, then taking all of the nodes from $G_2$, and taking a random permutation of those nodes. Then for each node in our permutation of $G_1$ we map that node to a node in our permutation of $G_2$ with no repitition. This will create a one-to-one and onto mapping from $G_1$ to $G_2$.
```

Comment:
You're only talking about the number of edges -- what about the rest of the definition? Each edge in the first graph must have a corresponding edge in the second graph.